src/components/App.jsx

```diff
@@ -7,10 +7,10 @@ import classNames from 'classnames'
 import Sidebar from './Sidebar'
 import Alerter from './Alerter'
 
-const App = ({ t, children }) => (
+export const App = ({ t, children }) => (
   <div className={classNames(styles['pho-wrapper'], styles['coz-sticky'])}>
 
-    <Sidebar />
+    <Sidebar t={t} />
```

Comment:
It is not how Cozy-Files-v3. I see your point but I am not sur if it's the right pattern to use. Again, I am curious about @goldoraf's opinion.

Edit: Is the goal to facilitate tests ? If so, I think this way is better.